### DISCUSSION TITLE
FMV - FRAM Model Visualiser - Version 3.1.0

### DISCUSSION THREAD

**Discussion by zerprize** [functionalresonance/FMV_Community_Edition#16] [Announcements]
This is now considered a stable release for general use, compiled and available to run in your browser from this github site:
[https://functionalresonance.github.io/FMV_Community_Edition/](https://functionalresonance.github.io/FMV_Community_Edition/)

Changes since the last announcement include: 
  - adding a report that can be opened in a separate browser window for sharing or printing as a PDF, in a format just like the old original installed application.
  - adding the Description field for Functions, just like the old original installed application. It will even display the descriptions from your old model files.
  - multiple selection of Functions, not just by 'lassoing', but also with ctrl+click on both the visual pane and the table view. You can also multiselect on the colour field in the table view. The colour of all selected functions can also be changed in bulk.
  - a new render called 'Greyscale', created specifically for publishing that requires vector graphics in black and white or greyscale. This render tries to balance the traditional look with some modern clarity.

In addition to all of the new features this version offers, this also concludes the transfer of all functionalities that existed in the old original versions. So, the migration and transformation are complete, formally closing a chapter on the FMV development history!

<hr /><em>This discussion was created from the release <a href='https://github.com/functionalresonance/FMV_Community_Edition/releases/tag/v3.1.0'>FMV - FRAM Model Visualiser - Version 3.1.0</a>.</em>

**Comment by dwityawaskito** [comment]
Hi, 
in this version, can you input the variability of the function?

**Reply by zerprize** [reply]
No, this version does not have the ability to input the variability of a function.
There is an extended project called the Analysis Sandbox that adds a metadata layer to the functions, where users can define their own data for each function. This includes the ability to define variability, or any other data, and to turn on the wavy line that graphically indicates a function that is exhibiting variability. 
This is currently a private repository available as a subscription option on the sponsorship page: [Sponsor](https://github.com/sponsors/functionalresonance?frequency=recurring)
